src/components/WriteButton/WriteButton.js

```diff
@@ -0,0 +1,12 @@
+import StyledWriteButton, { StyledAddImg } from "./styled";
+
+const WriteButton = (props) => {
```

Comment:
What do you think about destructuring the props? Something like:
```jsx
const WriteButton = ({ onClick }) => {
```
This way you avoid having to call `props` all the time. :smile: